ssas/service/public/api.go

```diff
@@ -292,24 +292,16 @@ func token(w http.ResponseWriter, r *http.Request) {
 	render.JSON(w, r, m)
 }
 
-func ValidateSecret(system ssas.System, secret string, w http.ResponseWriter, r *http.Request) (err error) {
+func ValidateSecret(system ssas.System, secret string, r *http.Request) (err error) {
```

Comment:
We could remove the `r http.request` parameter if we are not using it here!

Reply:
we would also want to update the return type to include the http status code that gets returned depending on the `if` conditionals. So you could do something like

```
if condition {
    err = errors.New(...)
    status = http.<status>
```

Reply:
@laurenkrugen-navapbc we are using `r http.request` on line 296. I considered returning the http status but it was exactly the same for each condition (http.statusunauthorized)